make walkthroughs 3scale 2.7 compatible (#115)
integr8ly/tutorial-web-app-walkthroughs: walkthroughs/2-fuse-aggregator-and-api-management/walkthrough.adoc

----

. Leave the *Description* field empty.
// The 'fuse-aggregation-app-url' should be the url of the Fuse Aggregation App e.g. https://fuse-flights-aggregator-ak49.cluster-lfa3xlh.opentry.me/
. In the *Private endpoint* field, enter:
+
[subs="attributes+"]
----
{route-fuse-flights-aggregator-host}
----
. Click *Create Backend* at the bottom of the screen.

=== Adding a new Product to Red Hat 3scale

> . Select *Dashboard* from the top navigation bar dropdown.

Reviewer: ```suggestion
. Select *Dashboard* from the top navigation menu.
```

> . From the *Dashboard*, in the *API* section, select the *Products* tab.

Reviewer: ```suggestion
. In the *API* section of the Dashboard, select the *Products* tab.
```

> . Select the *New Product* item.

Reviewer: ```suggestion
. Click *New Product*.
```

. Enter the following as the *Name* and *System name*:
+
[subs="attributes+"]
----
{fuse-flights-aggregator-app-name}
----
. Leave the *Description* field empty.

> . Click *Create Product* at the bottom of the screen.

Reviewer: don't we need a verification here?

Author: We can add one asking do they now see a the newly created product in the *APIs* section if that works for you?

=== Configuring Product in Red Hat 3scale

> . Click the *Settings* item from the *Integration* dropdown menu in the side navigation.

Reviewer: ```suggestion
. Select *Settings* from the *Integration* menu in the side navigation.
```
